### PR TITLE
Support native fonts again post-e10s.

### PR DESCRIPTION
**Pull request by pcwalton**
r? @glennw

### CODE REVIEW

File: components/gfx/font_cache_task.rs [hunk 1]
```diff
@@ -264,8 +264,13 @@ impl FontCache {
                     occupied.into_mut()
                 }
                 Entry::Vacant(vacant) => {
-                    let bytes = template.bytes();
-                    let font_key = webrender_api.add_raw_font(bytes);
+                    let font_key = match (template.bytes_if_in_memory(), template.ctfont()) {
```
Comment (glennw) [inline components/gfx/font_cache_task.rs]: Isn't this mac-specific code in a shared file?

**Comment by pcwalton** [comment]
Addressed comment. r? @glennw

**Comment by glennw** [comment]
@pcwalton Are the layout related commits included here deliberately?

**Comment by pcwalton** [comment]
@glennw Oops, will remove them

**Comment by pcwalton** [comment]
Fixed. r? @glennw

**Comment by glennw** [comment]
Thanks!

**Pull request merged**